### ISSUE TITLE
Passing custom Environment variables on Report Action

### ISSUE THREAD

**Issue by seang96** [flo-at/minmon#199]
I made a new Action for report that is a webhook and it requires passing an authentication header which I am storing as a secret. When I load it up with the new configuration, the action fails. Nginx logs a 403 status code so I assume the action isn't passing the environment variable to the header properly

Below is my report config sections

```
[[report.events]]
name = "Report"
action = "Ntfy Report"

[[actions]]
name = "Ntfy Report"
type = "Webhook"
url = "http://ntfy-svc.ntfy.svc.cluster.local/health"
body = "[{{env:MINMON_HOST}}] Monitoring uptime {{minmon_uptime_iso}} System uptime {{system_uptime_iso}}"
headers = {"Authorization" = "Bearer {{env:MINMON_NTFY_TOKEN}}"}
```

**Comment by flo-at** [comment]
Please try with just one pair of `{ }` instead of the double ones. Does it work that way?

**Comment by seang96** [comment]
Oh I figured it out...
- I set networking to host mode so I could monitor the computer's network interface, which in kubernetes means I can't use kubernetes networking anymore.
- When I tried using the external networking (actual domain name) it was getting 403 with the Authorization header because I forgot to swap it to https

Everything seems to be working in my kubernetes environment now. Thanks for the app, I am replacing Netdata monitoring with this!
